### PR TITLE
add split time field

### PR DESCRIPTION
**Pull request by Connor1996**
pd need to persist split time, and it is no need for tikv to fill this field.

### CODE REVIEW

File: proto/metapb.proto [hunk 1]
```diff
@@ -51,6 +51,7 @@ message Region {
     bytes end_key = 3;
     RegionEpoch region_epoch = 4;
     repeated Peer peers = 5;
+    int64 last_split_timestamp = 6;
```
Comment (disksing) [inline proto/metapb.proto]: Add some comment.

**Comment by Connor1996** [comment]
@huachaohuang If so, we can't guarantee not to merge new split region if pd leader change during importing. Is that acceptable?

**Comment by huachaohuang** [comment]
@Connor1996 I think that's acceptable if you don't merge regions in `split-merge-interval` after leader change.

**Comment by Connor1996** [comment]
@huachaohuang So persisting split time is still necessary, that is to say,  we can not just maintain the last split timestamp in memory.

**Comment by huachaohuang** [comment]
@siddontang @BusyJay Do you think we should add this to the region metadata?

**Pull request closed without merging**